code.go

```diff
@@ -221,6 +221,10 @@ func parseTags(repoName repos.RepoName, lines chan string) ([]*Code, error) {
 			continue
 		}
 		tag := parts[0]
+		if strings.HasPrefix(tag, "__anon") {
```

Comment:
I don't think you want to always do this from ctags.... but seeing as you're ditching ctags then it doesn't matter =)

Reply:
Why not actually? These are anonymous tags ctags creates when functions have no names.

Reply:
When I was investigating some "odd" behaviour of ctags before I noticed that the "operator[]" methods in dynamic_array were given __anon names. Nikolai said it was because "they technically didn't have a name".

Reply:
Yeah, but I realized that they are actually duplicated (causing also the failure in matrices.go), they were being detected as both `operator[]` and `__anon*`. I think the `__anon` tags appear because the `void operator[]() {}` declaration looks like a lambda.